hepdata/modules/records/templates/hepdata_records/components/resource_details.html

```diff
@@ -34,7 +34,14 @@ <h4>
       {% elif ctx.display_type == 'link' %}
         <p><a href="{{ ctx.resource.file_location }}">{{ ctx.resource.file_location }}</a></p>
       {% elif ctx.display_type == 'code' %}
-        <textarea id="code-contents" height="70%">{{ ctx.contents }}</textarea>
+
+        {% if ctx.contents %}
+          <textarea id="code-contents" height="70%">{{ ctx.contents }}</textarea>
+        {% else %}
+        <div id="code-contents-fail" height="70%">
+          <p style="font-weight:bolder">This file is larger than our loading threshold, and is only available for download below.</p>
```

Comment:
Can the file size (in MB) and the loading threshold (in MB) be displayed?  For example:

"This file (1.6 MB) is larger than our loading threshold (1.0 MB), and is only available for download below."